src/internal/hooks/useValue.ts

```diff
@@ -0,0 +1,5 @@
+import { useMemo } from 'react';
+
+export function useValue<T>(object: T): T {
+  return useMemo(() => object, [object]);
+}
```

Comment:
this gets rid of the huge `useMemo` block in context providers

Reply:
ok nice